### PR TITLE
D8CORE-747 Private page content type with a couple paragraphs

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Private page content type
- Private files paragraph type
- Text area paragraph type that only allows a wysiwyg without media embedding.

# Need Review By (Date)
- soon

# Urgency
- medium

# Steps to Test
1. checkout this branch
1. `blt drupal:install`
1. log in via `drush uli`
1. create a "Basic private page" content
1. ensure you only have access to 2 paragraph types
1. after creating it, ensure anonymous users can't view the page.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: .circleci/config/phpunit.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <!-- Example BROWSERTEST_OUTPUT_DIRECTORY value: /path/to/docrootroot/sites/simpletest/browser_output -->
     <env name="BROWSERTEST_OUTPUT_DIRECTORY" value="/var/www/html/docroot/sites/simpletest"/>
     <!-- To enable deprecation testing change `weak_vendors` to `max[self]=0`. -->
-    <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak_vendors"/>
+    <env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled"/>
```
Comment (sherakama) [inline .circleci/config/phpunit.xml:33]: What's this all about?

File: config/default/content_access.settings.yml [hunk 2]
```diff
@@ -9,3 +9,5 @@ content_access_roles_gids:
   stanford_student: 8
 _core:
   default_config_hash: 7LkiviGITIpniIOOIzpuxR7r4XzqqYN_cwu0fb0kx1M
+content_access_node_type:
+  hs_private_page: 'a:2:{s:4:"view";a:1:{i:0;s:13:"authenticated";}s:8:"per_node";i:1;}'
```
Comment (sherakama) [inline config/default/content_access.settings.yml:13]: This sure ain't pretty.

**Comment by sherakama** [comment]
That's a lot of new things. Do you need me to test the upgrade path as well?

File: .circleci/config/phpunit.xml (hunk 1 above)
Comment (pookmish) [reply .circleci/config/phpunit.xml:33]: Updated phpunit changes some of the variables and the way that it handles these. H&S needs to look into deprecation checking. this just isn't the way to do it.

File: config/default/content_access.settings.yml (hunk 2 above)
Comment (pookmish) [reply config/default/content_access.settings.yml:13]: right!? what were they thinking here. :man_shrugging:

**Comment by pookmish** [comment]
I think the upgrade path is tested via the behat tests. so it should be ok.

**Comment by pookmish** [comment]
@sherakama g2g?

**Review by sherakama — APPROVED** [review]
GTG

**Pull request merged**